R/parameters.R

```diff
@@ -24,5 +24,7 @@ get_parameters <- function(pop, dt = 1, time_period = 365, ...) {
 #'
 #' @param countryname name of country
 get_population <- function(countryname) {
-  squire::get_population(countryname, simple_SEIR = FALSE)
+  pop <- squire::get_population(countryname, simple_SEIR = FALSE)
```

Comment:
Don't think this is too important but are iso3c codes less ambigious than countrynames?

Reply:
I would be all over make the general switch to iso3c codes throughout. or either/or as we have in some of the squire functions.